javascript-timers/interval.js

```diff
@@ -0,0 +1,16 @@
+
+var $countdownDisplay = document.querySelector('.countdown-display');
+var $startingNumber = parseInt($countdownDisplay.textContent);
+
+function countdownDisplay() {
+  if ($startingNumber > 1) {
+    $startingNumber--;
+    $countdownDisplay.textcontent = $startingNumber;
+  } else {
+    $countdownDisplay.textContent = '~Earth Beeeelooowww Us~';
+    clearInterval(start);
```

Comment:
What value does `start` have? `clearInterval` expects the id of the interval to clear.

Reply:
clearInterval should be the intervId.

Reply:
Updated!